src/main/kotlin/io/github/mojira/arisa/modules/commands/ListUserActivityCommand.kt

```diff
@@ -36,7 +37,10 @@ class ListUserActivityCommand(
             )
         } else {
             issue.addRawRestrictedComment(
-                """No unrestricted comments from user "$sanitizedUserName" were found.""",
+                """No unrestricted comments from user "$sanitizedUserName" were found.
+                    |
+                    |Query: {{$jql}}
```

Comment:
The query includes the non-sanitized username so this defeats a bit the effort to sanitize it above.
Though including the non-sanitized username is most likely not that a big issue.

Reply:
Yeah, I'm aware this defeats the effort to sanitize the user name but a) I don't know if it's really too big of a deal... JIRA markdown is quite restricted, and it's not like you can just have a script tag as your username or something. I'd consider it to be more useful to know the exact query to debug it if necessary.